tests/app/helpers/test_schema_helper.py

```diff
@@ -97,6 +97,14 @@ def test_get_parent_options_for_block(self):
 
         self.assertEqual(parent_options, expected)
 
+    def test_get_duplicate_aliases(self):
+        survey_json = load_schema_file("test_repeating_household.json")
+
+        with self.assertRaises(AssertionError)as err:
+            aliases = SchemaHelper.get_aliases(survey_json)
+            for alias in aliases:
+                self.assertEquals(('Duplicate alias found: ' + alias)), err(err.exception)
```

Comment:
I'm not really sure what this is testing, you should need a survey with duplicate aliases to test the exception